website/docs/d/dns_managed_zone.html.markdown

```diff
@@ -0,0 +1,50 @@
+---
+layout: "google"
+page_title: "Google: google_dns_managed_zone"
+sidebar_current: "docs-google-datasource-dns-managed-zone"
+description: |-
+  Provides access to the attributes of a zone within Google Cloud DNS
+---
+
+# google\_dns\_managed\_zone
+
+Provides access to a zone's attributes within Google Cloud DNS.
+For more information see
+[the official documentation](https://cloud.google.com/dns/zones/)
+and
+[API](https://cloud.google.com/dns/api/v1/managedZones).
+```
+data "google_dns_managed_zone" "env_dns_zone" {
+  name        = "qa-zone"
+}
+
+resource "google_dns_record_set" "dns" {
+  name = "my-address.${data.google_dns_managed_zone.env_dns_zone.dns_name}"
+  type = "TXT"
+  ttl  = 300
+
+  managed_zone = "${data.google_dns_managed_zone.env_dns_zone.name}"
+
+  rrdatas = ["test"]
+}
+```
+
+## Argument Reference
+
+* `name` - (Required) A unique name for the resource, required by GCE.
```

Comment:
Oupps. I meant to clean up these descriptions before opening the PR... Forgot about it ;)